<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainWidget</class>
 <widget class="QWidget" name="TrainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelIllustration">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/res/pic/ok.svg</normaloff>:/pic/res/pic/ok.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNo">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/res/pic/cancel.svg</normaloff>:/pic/res/pic/cancel.svg</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
